usr/share/rear/layout/save/GNU/Linux/200_partition_layout.sh

```diff
@@ -26,7 +26,18 @@ extract_partitions() {
     declare sysfs_name=$(get_sysfs_name $device)
 
     ### check if we can find any partitions
-    declare -a sysfs_paths=(/sys/block/$sysfs_name/$sysfs_name*)
+    declare -a sysfs_paths_unfiltered=(/sys/block/$sysfs_name/$sysfs_name*)
+
+    ### initialize array
+    declare -a sysfs_paths=()
+
+    ### Silently skip invalid partitions on eMMC devices of type *rpmb or *boot[0-9],
```

Comment:
Could you add your explanation from
https://github.com/rear/rear/pull/2091#issue-262630887
to the comment directly to the code so that others who read the code at any time later
can directly understand why those partitions are invalid so that they can be silently ignored
and could you provide the full URL to the GitHub issue so that others who read the code
can directly copy and paste the URL into their web browser to view the GitHub issue,
for example a comment like:
```
### Silently skip invalid partitions on eMMC devices
### of type *rpmb or *boot[0-9] like /dev/mmcblk0rpmb or /dev/mmcblk0boot0
### because sysfs for (some?) eMMC disks recognises partitions,
### which are no usual partitions, but special areas on the eMMC.
### When trying to get the partition layout for such disks, ReaR would exit with an error.
### Therefore we ignore these special partitions, see also
### https://github.com/rear/rear/issues/2087
```

Reply:
Done :-)

Reply:
@fabz5 
thank you for your very prompt reaction!